config/initializers/constants.rb

```diff
@@ -0,0 +1 @@
+ADS_TIME = 60.minutes
```

Comment:
Are we sure ad times will always be constant?
Is there no possibility for different seances to have different ad times?

Reply:
Tobiasz told us to set it as constant, for example 30 minutes. But I also added time for cleaning the hall.

Reply:
I see, ok, no problem then :)